sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/hash.scala

```diff
@@ -635,6 +636,16 @@ case class HiveHash(children: Seq[Expression]) extends HashExpression[Int] {
   override protected def genHashBytes(b: String, result: String): String =
     s"$result = $hasherClassName.hashUnsafeBytes($b, Platform.BYTE_ARRAY_OFFSET, $b.length);"
 
+  override protected def genHashDecimal(
+      ctx: CodegenContext,
```

Comment:
They both aren't used but are a part of the method signature since the default impl in abstract class needs those : https://github.com/apache/spark/blob/3e40f6c3d6fc0bcd828d09031fa3994925394889/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/hash.scala#L321